### PR TITLE
{bio}[gompi/2019a] bmtagger v3.101

### PR DESCRIPTION
**Pull request by Darkless012**
(created using `eb --new-pr`)

requires ~~#9234~~ (SRPRISM)

### CODE REVIEW

**Comment by Darkless012** [comment]
Test report by @Darkless012
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3211.victini.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/407220102791be487306819c01812648 for a full test report.

File: easybuild/easyconfigs/b/bmtagger/bmtagger-3.101-foss-2019a.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+easyblock = 'MakeCp'
+
+name = 'bmtagger'
+version = '3.101'
+
+homepage = 'ftp://ftp.ncbi.nlm.nih.gov/pub/agarwala/bmtagger/'
+description = "Best Match Tagger for removing human reads from metagenomics datasets"
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```
Comment (boegel) [inline easybuild/easyconfigs/b/bmtagger/bmtagger-3.101-foss-2019a.eb]: @Darkless012 You can lower the toolchain to `gompi`?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/e8f667d7c20f0fc5ebb383fdddcf3806 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3141.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/d5ba1235f567e241038a0cdd7d702f71 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/b6d4092613de82fe90250e14b1376ce1 for a full test report.

**Comment by boegel** [comment]
re-triggering Travis...

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
easybuild.novalocal - Linux centos linux 7.7.1908, Intel Xeon Processor (Skylake, IBRS), Python 2.7.5
See https://gist.github.com/1bdf352a71babd170c9893bb765b8053 for a full test report.

**Comment by boegel** [comment]
Tests fail because the `SRPRISM` dep of `bmtagger` requires `Java` 1.8, while other easyconfigs using a `2019a` toolchain require Java 11...

So, it looks like we'll need to i) change the `NGS` dep for `SRPRISM` so it uses Java 11 as a dep, and ii) accept easyconfigs that depends on a specific version of Java as long as it's indicated in the `versionsuffix` (like I hinted to in #9158), see `check_dep_vars` in `test/easyconfigs/easyconfigs.py`.

Are you up for tackling that yourself @Darkless012 ?

**Comment by Darkless012** [comment]
> Are you up for tackling that yourself @Darkless012 ?
Sure I'm up :)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/b422fc61bfb8d80c3466177da32f2f85 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3200.victini.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/ba30c463af340081e0489e32f6e4a326 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @Darkless012!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2604.swalot.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/7aad85d5d6193315c840650c98b2a900 for a full test report.

**Pull request merged**